src/ucp/api/ucp.h

```diff
@@ -631,6 +633,120 @@ void ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WAKEUP
+ * @brief Obtain an event file descriptor for event notification.
+ *
+ * This routine returns a valid file descriptor for polling functions.
+ * The file descriptor will get signaled when an event occurs, as part of the
+ * wake-up mechanism. Signaling means a call to poll() or select() with this
+ * file descriptor will return at this point, with this descriptor marked as the
+ * reason (or one of the resons) the function has returned.
+ *
+ * The worker must be armed before waiting on an event (must be re-armed after
+ * it has been signaled for re-use) with @ref ucp_worker_arm.
+ * The events triggering a signal of the file descriptor depend on the
+ * interfaces used by the worker and are defined in the transport layer, and
+ * typically represent a request completion or newly available resources.
+ * The file descriptor should be closed by invoking close().
```

Comment:
is this the wording you're suggesting to replace "The file descriptor should be closed by invoking close()."?